glide.yaml

```diff
@@ -27,26 +28,35 @@ import:
   subpackages:
   - github
 - package: github.com/inconshreveable/go-update
+- package: github.com/mitchellh/mapstructure
+  version: db1efb556f84b25a0a13a04aad883943538ad2e0
 - package: github.com/kardianos/osext
 - package: github.com/olekukonko/tablewriter
 - package: github.com/pkg/browser
+# latest viper have issue with IsSet: https://github.com/spf13/viper/issues/276
 - package: github.com/spf13/viper
-  repo: https://github.com/jimmidyson/viper.git
-  vcs: git
+  version: 382f87b929b84ce13e9c8a375a4b217f224e6c65
 - package: github.com/xeipuuv/gojsonschema
-- package: golang.org/x/crypto
-  subpackages:
-  - ssh
-- package: gopkg.in/cheggaaa/pb.v1
+- package: github.com/blang/semver
+- package: github.com/docker/go-units
 - package: github.com/pkg/errors
   version: ^0.8.0
 - package: github.com/jteeuwen/go-bindata
   version: ~3.0
+- package: github.com/spf13/cobra
+  subpackages:
+  - doc
+- package: github.com/spf13/pflag
+- package: golang.org/x/crypto
+  subpackages:
+  - ssh
+- package: golang.org/x/oauth2
+- package: gopkg.in/cheggaaa/pb.v1
 # The following repositories need to be specified explicitly, since the versions referenced by the openshift/origin dependency
 # refer to forks of these projects within the openshift organizaton
 # This needs to be manually updated after upgrading the openshift/origin dependnecy
 - package: k8s.io/kubernetes
-  version: 52492b4bff99ef3b8ca617d385a3ff0612f9402d
+  version: a9e9cf3b407c1d315686c452bdb918c719c3ea6e
```

Comment:
hopefully we can get soon rid of this dependency now. It takes most time to checkout and compile and is useless for us